src/webchat-ui/components/presentational/chat-options/ChatOptionsFooter.tsx

```diff
@@ -29,13 +30,20 @@ const StyledFooterTypography = styled(Typography)(() => ({
 	margin: 0,
 }));
 
+interface IChatOptionsFooterProps {
+	settings: IWebchatSettings;
+}
+
 // TODO: Get text and URL for the footer items from the config
-export const ChatOptionsFooter = () => {
-	const footerItem1Text = "Imprint";
-	const footerItem2Text = "Data Protection";
+export const ChatOptionsFooter = (props: IChatOptionsFooterProps) => {
+	const { settings } = props;
+	const { chatOptions } = settings;
+
+	const footerItem1Text = chatOptions?.footer?.items[0]?.title || "Imprint";
+	const footerItem2Text = chatOptions?.footer?.items[1]?.title || "Data Privacy";
 
-	const footerItem1URL = "https://www.cognigy.com/legal-notice";
-	const footerItem2URL = "https://www.cognigy.com/privacy-policy";
+	const footerItem1URL = chatOptions?.footer?.items[0]?.url || "https://www.cognigy.com/legal-notice";
+	const footerItem2URL = chatOptions?.footer?.items[1]?.url || "https://www.cognigy.com/privacy-policy";
 
```

Comment:
Can you please add a condition (chatOptions?.footer?.items[1])  in line 60 to display the second link?